tljh/installer.py

```diff
@@ -275,6 +275,24 @@ def ensure_jupyterhub_running(times=4):
     raise Exception("Installation failed: JupyterHub did not start in {}s".format(times))
 
 
+def ensure_symlinks(prefix):
+    """
+    Ensure we symlink appropriate things into /usr/local/bin
+
+    We add the user conda environment to PATH for notebook terminals,
+    but not the hub venv. This means tljh-config is not actually accessible.
+
+    We symlink to /usr/local/bin to 'fix' this. /usr/local/bin is the appropriate
+    place, and works with sudo -E
+    """
+    tljh_config_src = os.path.join(prefix, 'bin', 'tljh-config')
+    tljh_config_dest = '/usr/local/bin/tljh-config'
+    if not os.path.exists(tljh_config_dest):
+        # If this exists, we leave it alone. Do *not* remove it,
+        # since we are running as root and it could be anything!
+        os.symlink(tljh_config_src, tljh_config_dest)
```

Comment:
If the file exists, probably worth checking the link and warning about it?

Possible cases to test:

- symlink exists and already points to our tljh-config
- symlink exists and points to another tljh install that exists (may not work due to version mismatch)
- symlink exists but target doesn't (definitely won't work)

Reply:
So:

1. Symlink exists and points to our tljh-config - do nothing.
2. *something* else exists - warn, but don't abort?
3. symlink exists but target doesn't - force our own symlink?

Reply:
Yeah, I think that's sensible

Reply:
@minrk I think I implemented a variant of it now.